packages/react/src/components/List/List.test.jsx

```diff
@@ -157,4 +158,41 @@ describe('List', () => {
     userEvent.click(screen.getByTitle('Item 1'));
     expect(onSelect).not.toHaveBeenCalled();
   });
+  it('calls handleLoadMore when load more row clicked', () => {
+    const mockLoadMore = jest.fn();
+    render(
+      <List
+        title="Sports Teams"
+        items={[
+          {
+            id: 'org',
+            content: { value: 'Organization' },
+            children: [
+              { id: 'site-01', content: { value: 'Site 1' } },
+              {
+                id: 'site-02',
+                content: { value: 'Site 2' },
+                children: [
+                  { id: 'system-01', content: { value: 'System 1' } },
+                  { id: 'system-02', content: { value: 'System 2' } },
+                ],
+                hasLoadMore: true,
```

Comment:
so each child element has to have a property that identifies that the children can be loaded?  So I could set hasLoadMore on every item?

Reply:
yep, I would say mostly for the hierarchy table because the idea behind it is that I would not load all children of that, so I could have "paging"  for that child, where the load of it happens during hierarchy expansion, and I don't have all the data prior loaded.